bitnami/redmine/values.yaml

```diff
@@ -470,3 +470,17 @@ mailReceiver:
   ## Defines a list of attributes which can be overrided by received email body
   ## "all" means that all attributes can be overrided
   allowOverride: ""
+
+# Add custom certificate authorities to redmine container
+certificates:
+  customCA: []
+  # - secret: custom-CA
+  # - secret: more-custom-CAs
+  image:
+    repository: alpine
+    # pullPolicy:
+    # pullSecrets
+    #   - myRegistryKeySecretName
```

Comment:
Could you modify this following the standard binami image structure in `values.yaml`? Please take a look at https://github.com/bitnami/charts/tree/master/bitnami/common#imageroot

Also, please use `minideb` as https://github.com/bitnami/charts/blob/master/bitnami/ghost/values.yaml#L42

Reply:
These have been added along with the ability to use custom certificates